commit 25ab3d350e362fb10ccdef49b23d60f95e88460c
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-08-22T20:57:34+01:00 (7 weeks ago)
    
    TOOLS-2247 move sdcnode to manta (fix Makefile)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index ae9e2b0..b77f3eb 100644
--- a/Makefile
+++ b/Makefile
@@ -48,7 +48,7 @@ nodes: nodesrc
 
 .PHONY: publish
 publish: prepublish
-	rm -rf $(TOP)/bits
+	rm -rf $(TOP)/bits/sdcnode
 	mkdir -p $(TOP)/bits/sdcnode
 	cp $(TOP)/build/nodes/*/sdcnode-*.tgz $(TOP)/bits/sdcnode
 
